dataset-spec/template.schema.json

```diff
@@ -142,88 +204,123 @@
-                                    "type": "object",
-                                    "additionalProperties": true,
-                                    "required": [
-                                        "data_url",
-                                        "license_url"
-                                    ],
-                                    "properties": {
-                                        "home_url": {
-                                            "type": "string",
-                                            "format": "uri",
-                                            "examples": [
-                                                ["https://resources.marine.copernicus.eu/"]
-                                            ]
-                                        },
-                                        "data_url": {
-                                            "type": "string",
-                                            "format": "uri",
-                                            "examples": [
-                                                ["https://resources.marine.copernicus.eu/"]
-                                            ]
-                                        },
-                                        "license_url": {
-                                            "type": "string",
-                                            "format": "uri",
-                                            "examples": [
-                                                ["https://marine.copernicus.eu/user-corner/service-commitments-and-licence"]
-                                            ]
-                                        },
-                                        "attributions": {
-                                            "type": "array",
-                                            "items": {"type":  "string"}
-                                        },
-                                        "processing_steps": {
-                                            "type": "array",
-                                            "description": "processing steps to be applied to the source",
-                                            "items": {"type":  "string"},
-                                            "examples": [
-                                                [
-                                                    "Masking out low quality pixels",
-                                                    "Temporal NaN-mean aggregation",
-                                                    "Spatial bi-cubic interpolation"
-                                                ]
+                            "metadata": {
+                                "type": "object",
+                                "required": [
+                                    "color_bar_name",
+                                    "color_value_min",
+                                    "color_value_max"
+                                ],
```

Comment:
Afaik, these were SHOULDs aka recommendations.

Reply:
The current specification says:

> The following variable `metadata` properties are **mandatory**, because
> we want a defined way to display image tiles: 
> 
> ```json
>   "metadata": {
>     ...,          
>     "color_bar_name": "YlGn",
>     "color_value_min": 0.5
>     "color_value_max": 150
> ```

Should I leave as is, or change both specification and schema to make them optional?

Reply:
Leaving as is.